README.md

```diff
@@ -31,12 +31,59 @@ cd dianna
 python3 -m pip install -e .
 ```
 
+
 ## Documentation [![Documentation Status](https://readthedocs.org/projects/dianna/badge/?version=latest)](https://dianna.readthedocs.io/en/latest/?badge=latest)
 
+### Benchmark datasets
+(TODO: add a sentence and/or image for each and the Zenodo and/or original links and links to notebooks for creating them)
+
+DIANNA offers to use simple benchamrk datasets for evaluating and comparing the XAI methods:
+
+#### Images
+* Binary (2-class) MNIST
+* Simple Geometric (triangles and circles)
+* Simple Scientific (LeafSnap30)
+
+#### Text
+* Movie reviews treebank
+
+### ONNX models
+(TODO: Add all links) 
+
+**We work with ONNX!** ONNX is a great unified oen NN standard which can be used to boost reproducible science. Your ONXX model also gives you boost in performance! In case your models are still in another popular DNN format, here are the simple recipies on to convert them:
```

Comment:
```suggestion
**We work with ONNX!** ONNX is a great unified NN standard which can be used to boost reproducible science. Using ONXX for your model also gives you a boost in performance! In case your models are still in another popular DNN format, here are some simple recipes to convert them:
```